### ISSUE TITLE
Does not activate radio items

### ISSUE THREAD

**Issue by fkloft** [p-e-w/plotinus#24]
Noticed this in Totem. It seems radio menu items are not selected by plotinus.

Steps to reproduce:
* Open a video with multiple audio streams or subtitles in totem
* Activate plotinus, search for a language
* Select command to switch audio or subtitle.

Expected result:
The selected audio stream or subtitle is enabled.
Actual result:
The previously active stream is still enabled.

Checkboxes however are not affected. Totem has a checkbox to "Zoom in", which can be toggled with plotinus.

When trying to select a radio item (which seems to be a GtkAction, but not a GtkToggleAction), the following message appears in the terminal:
```
(totem:20137): GLib-GIO-CRITICAL **: g_simple_action_activate: assertion 'simple->parameter_type == NULL ? parameter == NULL : (parameter != NULL && g_variant_is_of_type (parameter, simple->parameter_type))' failed
```

**Comment by p-e-w** [comment]
Fixed! Thank you for reporting this important issue!

**Comment by fkloft** [comment]
Confirmed fixed as of 2c2a711ad137b2eedc26f790b0150ce3f157ec50.

Thank you very much!
